packages/scandipwa/src/route/Checkout/Checkout.container.js

```diff
@@ -377,14 +380,62 @@ export class CheckoutContainer extends PureComponent {
         return true;
     };
 
-    containerProps = () => {
-        const { paymentTotals } = this.state;
+    containerProps() {
+        const {
+            cartTotalSubPrice,
+            history,
+            isEmailAvailable,
+            isMobile,
+            setHeaderState,
+            totals,
+            isInStoreActivated
+        } = this.props;
+        const {
+            billingAddress,
+            checkoutStep,
+            email,
+            estimateAddress,
+            isCreateUser,
+            isDeliveryOptionsLoading,
+            isGuestEmailSaved,
+            isLoading,
+            orderID,
+            paymentMethods,
+            paymentTotals,
+            selectedShippingMethod,
+            shippingAddress,
+            shippingMethods,
+            selectedStoreAddress,
+            isPickInStoreMethodSelected
+        } = this.state;
```

Comment:
We **MUST** do something about this.

This is an unacceptable amount of props and state properties forwarded to the component.